cairo_zero/kakarot/constants.cairo

```diff
@@ -37,6 +37,9 @@ namespace Constants {
     // FIELD PRIME
     const FELT252_PRIME_HIGH = 0x8000000000000110000000000000000;
     const FELT252_PRIME_LOW = 0x1;
+
+    // Infinite gas handle_l1_message
+    const INFINITE_GAS = 2100000000;
```

Comment:
let's try `P - 1`

Reply:
let's use `2**64 - 1` instead, as in most clients gas is a u64 field

Reply:
Done